eng/generate.ps1

```diff
@@ -32,7 +32,7 @@ function Get-Help() {
   Write-Host "                                               option above, one per line. If specified, the -package option is ignored."
   Write-Host "  -d|-destination                              A path to the root of the repo to copy source into."
   Write-Host "  -t|-type                                     Type of the package to generate. Accepted values: ref (default) | text."
-  Write-Host "  -e|-excludeDependencies                      Determines if package dependencies should be excluded. Default is false."
+  Write-Host "  -excludeDependencies                         Determines if package dependencies should be excluded. Default is false."
```

Comment:
I personally use the aliases in my local dev workflows.  I utilize this option frequently.  Consider changing the alias to a different alias such as 'x'.

Reply:
Updated and tested locally. "-x" works as expected.